recipes/gmp/config.yml

```diff
@@ -1,5 +1,7 @@
 versions:
-  "6.1.2":
-    folder: all
   "6.2.0":
     folder: all
+  "6.2.1":
+    folder: all
```

Comment:
```suggestion
  "6.2.1":
    folder: all
  "6.2.0":
    folder: all
```

Reply:
Is it work adding `6.2.0` ... maybe justy keep the point release?

Reply:
I add 6.2.1 in this pr.
This comment was just about ordering the versions in descending order.